js/basicblocks.js

```diff
@@ -1519,14 +1519,14 @@ function initBasicProtoBlocks(palettes, blocks, beginnerMode) {
         newblock.hidden = true;
     }
 
-    var newblock = new ProtoBlock('notecounter');
+    var newblock = new ProtoBlock('sumnotevalue');
```

Comment:
You should change the label, but not the protoblock name, since that will break existing projects.